index.js

```diff
@@ -1,20 +1,23 @@
-require('dotenv').config()
+require('dotenv').config();
 const express = require('express');
 const path = require('path');
 const cors = require('cors');
 const app = express();
-const Henesis = require('@haechi-labs/henesis-sdk-js').default;
+const { TransactionTracker } = require('@haechi-labs/henesis-sdk-js');
 const { Sender } = require('./helper/Sender');
 
 const transactions = {};
 
 const { CLIENT_ID, PRIVATE_KEY, NODE_ENDPOINT } = process.env;
-const henesis = new Henesis(CLIENT_ID);
+const tracker = new TransactionTracker(CLIENT_ID, {
```

Comment:
I write as `transactionTracker` not `tracker` in `docs`. you prefer `tracker`?

Reply:
Yes. I prefer short names.